Makefile

```diff
@@ -121,9 +122,15 @@ git-commit:
 	git add catalog-manifests/ manifests/
 	git commit -m "New catalog: $(CATALOG_VERSION)" --author="OpenShift SRE <aos-sre@redhat.com>"
 
+.PHONY: git-tag
+.SILENT: git-tag
+git-tag:
+	# attempt to tag, do not recreate a tag (only happens if changes happen outside of catalog-manifests/)
+	git tag $(GIT_TAG) 2> /dev/null && echo "INFO: created tag: $(GIT_TAG)" || echo "INFO: git tag already exists, skipping tag creation: $(GIT_TAG)"
+
 .PHONY: git-push
-git-push:
-	git push
+git-push: git-tag
+	git push && git push --tags
```

Comment:
We should take care to ensure the content ends up in openshift/osd-operators-registry and not a fork's remote?

Reply:
If the tracked branch is a fork it will push tags to that fork.  That's fine, I don't want to have random fork's tags in the upstream repo.

Reply:
Seems reasonable.